aicsimageio/readers/tiff_reader.py

```diff
@@ -130,7 +130,10 @@ def _read_delayed(self) -> da.core.Array:
             # Combine length of scenes and operating shape
             # Replace YX dims with empty dimensions
             operating_shape = (len(scenes), *operating_shape)
-            operating_shape = operating_shape[:-2] + (1, 1)
+            if scenes[0].keyframe.samplesperpixel != 1:  # if it's a multichannel (RGB)
```

Comment:
Can you describe a bit more how this handles the XYS case?

Reply:
As I understand it if samplesperpixel is anything except 1 it means that there are multiple "Channels" encoded per pixel, like 24 bit RGB => 3x gray8 channels. 

The prior code essentially tells dask to read (Y, X) planes by replacing them with (1, 1) 
I just tweeked it to say read the (Y, X, S) by replacing the last three with (1, 1, 1). 

[Did you mean to explain this here or as a comment in the code?]

Reply:
Explaining it here was fine! I just needed a bit more context, thanks 🙂